### DISCUSSION TITLE
Transfer server

### DISCUSSION THREAD

**Discussion by reebokHCFR** [photoprism/photoprism#1755] [Q&A] [answered]
My main computer (windows) is light years beyond my laptop pihole/plex server which is Ubuntu server. Can I install photoprism on my main computer to do all the indexing, then move it over to the ubnutu server when all the heavy lifting is done? Thanks.

**Comment by lastzero** [comment, marked as answer]
Yes, but manual steps are required, such as copying files or creating an index backup. So it depends on whether the time saved is worth the work.
